torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -110,6 +111,14 @@ bool isSimpleMap(Node *node) {
   return true;
 }
 
+Value * broadcastSizes(at::ArrayRef<Value*> sizes) {
```

Comment:
nit: This might be good to have in SymbolicVariable

Reply:
On one hand yes, on the other hand it's not necessary anywhere else at the moment. Also, the inputs are *not tensors* (they are sizes), so it's not really a *Variable anymore (it's more of a *Value).

Reply:
True, that's a good point